src/db/services/verification.ts

```diff
@@ -0,0 +1,28 @@
+import { Document } from "mongoose";
+
+import { BaseService } from "./base";
+import { Verification } from "../schemas";
+// import { IVerification } from "../types";
```

Comment:
does this need to be commented out pt 1 the miracle and the sleeper

Reply:
done